pkg/client/cache/fifo.go

```diff
@@ -35,6 +35,8 @@ type FIFO struct {
 	// keyFunc is used to make the key used for queued item insertion and retrieval, and
 	// should be deterministic.
 	keyFunc KeyFunc
+	// isOlder is used to accept or reject objects which are older than what's already stored.
+	isOlder IsOlder
```

Comment:
it's not used to accept, so strike that...  but also don't we want to reject things that are equally as old?  no point in accepting it since it's just wasted effort.